### ISSUE TITLE
Could not find module "@angular-builders"

### ISSUE THREAD

**Issue by jogelin** [angular-builders/jest#2]
I followed your instructions and configure `angular.json` but when I ran `ng test appName` I have this error:

```
Could not find module "@angular-builders" from "C:\\Users\\.....\\dev\\....\\..\\monorepo".
....

```

The module is well installed in `node_modules/@angular-builders`

**Comment by jogelin** [comment]
Nervermind :)

**Comment by meltedspark** [comment]
Yep, a typo in README. Instead of `"builder": "@angular-builders:jest:run"` it should be 
`"builder": "@angular-builders/jest:run"`.
It's also not specified in the README that you have to tweak _tsconfig.spec.json_ a bit. You have to change `module` to `commonjs`:
```
  "compilerOptions": {
    ...
    "module": "commonjs"
  }
```
As jest is caching files, it won't recognize your changes immediately. You can disable it (just for one time) by specifying `"no-cache": true` in jest builder options in _angular.json_.

**Comment by meltedspark** [comment]
I think you should look into [testMatch](https://jestjs.io/docs/en/configuration.html#testmatch-array-string) option in _jest.config.js_.  
Anyways, please keep me up to date on your progress, because multi-app workspace is a common use case for Angular 6 and I want to be sure the builder does fit it.  
I'll leave this one open for fixing the README.
If you have any problems or usability requests please don't hesitate opening issues.

**Comment by jogelin** [comment]
Tomorrow I'll propose an update of your example app with a good configs !

**Comment by meltedspark** [comment]
Great, can't wait to see that!
